site/en/docs/devtools/issues/index.md

```diff
@@ -16,11 +17,12 @@ content.
 
 {% Aside %}
 
-In Chrome 84, the Issues tab supports three types of issue:
+Starting from Chrome 92, the **Issues** tab supports the following types of issues:
 
 - [Cookie problems][1]
 - [Mixed content][2]
 - [COEP issues][3]
+- [CORS errors][6]
```

Comment:
iirc, the Issues tab now support more than the listed types, we don't announce all the new issue types in WNDT because it's not significant changes.

Check with the team on the types or "and more.."

Reply:
Thanks for the clarification. I'll figure out which types are missing and add them in a new PR.